### PR TITLE
Update csa.md to remove non-required parameters

### PR DESCRIPTION
**Pull request by mmcao**
Removed non-required parameters (width, layout) from the amp-ad tag.

### CODE REVIEW

**Comment by mmcao** [comment]
@megankj can you review?

File: ads/google/csa.md [hunk 1]
```diff
@@ -45,9 +45,9 @@ amp-ad tag.  The values you pass should be set to a stringified version of the
 Javascript object you would pass in the ad request of a standard CSA request.
 
 ```html
-<amp-ad width=auto height=300
+<amp-ad 
+    height=300
     type='csa'
-    layout='fixed-height'
     data-afsh-page-options='{"pubId": "partner-vert-pla-pubid-pdp",
     "query": "flowers"}'
```
Comment (megankj) [inline ads/google/csa.md]: can this fit on one line?

File: ads/google/csa.md [hunk 2]
```diff
@@ -69,9 +69,9 @@ not return ads, AMP will request AFS ads with the values from the **data-afs-***
 attributes.
 
 ```html
-<amp-ad width=auto height=400
+<amp-ad 
+    height=400
     type='csa'
-    layout='fixed-height'
     data-afsh-page-options='{"pubId": "partner-vert-pla-pubid-pdp",
     "query": "flowers"}'
```
Comment (megankj) [inline ads/google/csa.md]: can this fit on one line?

**Comment by megankj** [comment]
Can this PR be merged?

**Pull request merged**